Resources/Prototypes/Nyanotrasen/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -32,3 +32,5 @@
     modifiers:
       coefficients:
         Caustic: 0.75
+  - type: ToggleableClothing
+    clothingPrototype: ClothingHeadHoodMysta
```

Comment:
From reading into this, I think this actually needs to be parented to ClothingOuterStorageToggleableBase instead of CltohingOuterStorageBase. The latter example doesn't have a correct container setup for storing the hood.